__tests__/app.test.js

```diff
@@ -296,3 +296,29 @@ describe("DELETE: /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET: /api/users", () => {
+  test("GET 200: to respond with an array of object containing the users with their username, usr and avatar_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.users)).toBe(true);
```

Comment:
No need to check for it being an array here as we are later proving its an array as we're using a forEach